sidebar_translations.json

```
@@ -28,7 +28,7 @@
   "develop.items.food": "Food Items",
   "develop.items.custom_armor": "Custom Armor",
   "develop.items.custom_tools": "Custom Tools",
-  "develop.items.custom_item_groups": "Custom Item Groups",
+  "develop.items.custom_item_groups": "Custom Creative Tabs",
```

```suggestion
  "develop.items.custom_creative_tabs": "Custom Creative Tabs",
```

and change `.vitepress/sidebars/develop.ts` too